src/client/src/error.rs

```diff
@@ -25,18 +25,8 @@ pub enum Error {
         backtrace: Backtrace,
     },
 
-    #[snafu(display(
-        "Failed to do Flight get, addr: {}, code: {}, err_msg: {}",
-        addr,
-        code,
-        err_msg
-    ))]
-    FlightGet {
-        addr: String,
-        code: u32,
-        err_msg: String,
-        backtrace: Backtrace,
-    },
```

Comment:
I split this error into two part. The new `FlightGet` describes an rpc call fail, with root cause and server address. `ExternalError` is a temporary error container for assembling errors from flight metadata. And the call stack of both is clear so I remove the backtrace.